substratest/assets.py

```diff
@@ -21,6 +21,11 @@ def get(self):
         raise NotImplementedError
 
 
+class TupleStatus(Enum):
+    done = auto()
+    failed = auto()
```

Comment:
That's good! We could defined all the statuses here: 'todo', 'doing', 'waiting' and 'failed' (list to validate).

Reply:
Absolutely, I'll do that!